src/core/agent/records/operationPendingRecord.type.ts

```diff
@@ -0,0 +1,3 @@
+export enum OperationPendingRecordType {
+  IDENTIFIER = "Identifier",
```

Comment:
Lets match KERIA here: https://github.com/WebOfTrust/keria/blob/a1a9b02ceda45fb83885f8ad97cc8de78cb96e45/src/keria/core/longrunning.py#L24

What type of op do we have here? It's the first part of the operation ID. Is it "witness"?

Reply:
@iFergal Yes. I will change. Because I think OperationPendingRecordType is type of record in IDW database